save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/kubernetes/KubernetesPersistentVolumeService.kt

```diff
@@ -0,0 +1,40 @@
+package com.saveourtool.save.orchestrator.kubernetes
+
+import com.saveourtool.save.orchestrator.service.PersistentVolumeService
+import io.fabric8.kubernetes.api.model.HostPathVolumeSource
+import io.fabric8.kubernetes.api.model.PersistentVolume
+import io.fabric8.kubernetes.api.model.PersistentVolumeSpec
+import io.fabric8.kubernetes.client.KubernetesClient
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+import java.nio.file.Path
+import kotlin.io.path.absolutePathString
+import kotlin.io.path.copyTo
+import kotlin.io.path.createTempDirectory
+
+/**
+ * Implementation of [PersistentVolumeService] that creates Persistent Volumes in Kubernetes
+ */
+@Profile("kubernetes")
+@Component
+class KubernetesPersistentVolumeService(
+    private val kc: KubernetesClient,
+) : PersistentVolumeService {
+    override fun createFromResources(resources: Collection<Path>): KubernetesPvId {
+        val tmpDir = createTempDirectory()
```

Comment:
maybe create temp folder only inside `configProperties.repository`?

Reply:
I believe it will be created under `/tmp` and it'll make it easier to clean up. Also, I'm planning to implement this for Kubernetes only in subsequent PRs, so I probably will use something better than hostPath mount anyway

Reply:
it will be created under `/tmp`, but we run in docker. do we have access to `/tmp`?

Reply:
I was initially planning to copy files into a temporary dir on the node and mount in as `hostPath` to the pod. However I'll use a proper persistent volumes in the next PR